awesomemotive/file-upload-types
# restricting file upload types

Fixes #35 

- adds file types natively supported by WP to the Settings > File upload types table so user can now disable them if needed

Test steps:
- activate fut plugin
- go to wp-admin > settings > file upload types and check if file types with description "WordPress natively registered type" are added and automatically enabled
- disable some poplar native file type (for example .jpeg) and save
- install wpforms and create form with file upload field
- preview form and try to upload .jpeg file -> you should see error file extension is not supported
- go to  wp-admin > settings > file upload types and re-enable .jpeg file extension
- go to form and try to upload .jpeg file -> you should be able to do this
